tests/cases/standalone/common/information_schema/region_statistics.sql

```diff
@@ -0,0 +1,25 @@
+USE public;
+
+CREATE TABLE test (
+    a int primary key,
+    b string,
+    ts timestamp time index,
+) PARTITION ON COLUMNS (a) (
+    a < 10,
+    a >= 10 AND a < 20,
+    a >= 20,
+);
+
+
+INSERT INTO test VALUES
+       (1, 'a', 1),
+       (11, 'b', 11),
+       (21, 'c', 21);
+
+-- SQLNESS SLEEP 11s
+-- FIXME(dennis): we need to wait the datanode reporting stats info to metasrv.
```

Comment:
@WenyXu It's advisable to adjust the heartbeat frequency for region stats information; it's currently set to 10 by default, but we can reduce it for faster test execution.